### DISCUSSION TITLE
The new vision Q: How to output PLY file?

### DISCUSSION THREAD

**Discussion by choisbody** [thygate/stable-diffusion-webui-depthmap-script#199] [Q&A]
When i try to "Generate 3D inpainted mesh. (Sloooow)" , Then model is OBJ file. How to output PLY file like old vision?(old vision can't used myself depthmap photo).
Thanks for answer~

**Comment by thygate** [comment]
in the webui settings tab, enable 'additional PLY file' under the depthmap settings...

**Reply by choisbody** [reply]
Thanks for your answer~ Nice work ~
